app/components/subject-viewer.cjsx

```diff
@@ -158,6 +158,24 @@ module.exports = createReactClass
                   <button aria-label="Play" title="Play" type="button" className="secret-button subject-tools__play" onClick={@setPlaying.bind this, true}>
                     <i className="fa fa-play fa-lg fa-fw"></i>
                   </button>}
+                <div className="subject-frame-duration">
+                  <i className="fa fa-angle-right fa-lg fa-fw subject-frame-duration--slower"></i>
+                  <input
+                    aria-valuemin="-2000"
```

Comment:
`aria-valuemin` is only needed on elements that don't support the `min` attribute.